File: driver/lib/models/trip.dart
import 'package:cloud_firestore/cloud_firestore.dart';

class Trip {
  final String id;
  final bool active;
  final DateTime? actualArrivalTime;
  final DateTime? actualDepartureTime;
  final GeoPoint? currentLocation;
  final int capacityInVehicle;
  final int? passengersQty;
  final String? driverId;
  final DateTime intendedArrivalTime;
  final DateTime intendedDepartureTime;
  final String routeId;
  final String? vehicleId;

  Trip ({
    required this.id,
    required this.active,
    this.actualArrivalTime,
    this.actualDepartureTime,
    this.currentLocation,
    required this.capacityInVehicle,
    this.passengersQty,
    this.driverId,
    required this.intendedArrivalTime,
    required this.intendedDepartureTime,
    required this.routeId,
    this.vehicleId,
  });

}

File: web/firebase/fireapp/models/trip.py
from django.db import models

class Trip(models.Model):
    Id = models.CharField(max_length=200)
    ActualArrivalTime = models.DateTimeField()
    ActualDepartureTime = models.DateTimeField()
    DriverId = models.CharField(max_length=200)
    IntendedArrivalTime = models.DateTimeField()
    IntendedDepartureTime = models.DateTimeField()
    PassengersQty = models.IntegerField()
    RouteId = models.CharField(max_length=200)
    VehicleId = models.CharField(max_length=200)
    CapacityInVehicle = models.IntegerField()